config/env/default.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
   domain: process.env.DOMAIN || 'localhost:3000',
   supportEmail: 'support@trustroots.org', // TO-address for support requests
   profileMinimumLength: 140, // Require User.profile.description to be >=140 chars to send messages
+  // A message below longMessageMinimumLength will be 'short' in influxdb, otherwise 'long'
+  // TODO choose an expressive value
+  // Used in ./modules/messages/server/services/message-to-influx.server.service
+  longMessageMinimumLength: 150,
```

Comment:
I had a look at the DB and from among initial messages:
- < 60 chars: 4%
- < 100 chars: 8%
- < 150 chars: 14% 
- < 160 chars: 16%
- < 170 chars: 17%
- < 180 chars: 19%
- < 190 chars: 20%
- < 200 chars: 21%
- < 210 chars: 23%

I went trough my TR inbox and less than 200 chars still feels pretty short to me.

Let's settle to 170 chars work on getting that 17% way down.